pkgs/applications/networking/mailreaders/aerc/default.nix

```diff
@@ -14,7 +14,7 @@ buildGoModule rec {
   };
 
   runVend = true;
-  vendorSha256 = "0avdvbhv1jlisiicpi5vshz28a2p2fgnlrag9zngzglcrbhdd1rn";
+  vendorSha256 = "13zs5113ip85yl6sw9hzclxwlnrhy18d39vh9cwbq97dgnh9rz89";
```

Comment:
Where does these checksum changes come from?

Reply:
Upstreams implementation of replacements seems to handle duplicates in the vendor folder.